src/dcqc/tests/base_test.py

```diff
@@ -195,12 +196,22 @@ def _interpret_process_outputs(self, outputs: dict[str, Path]) -> TestStatus:
         """Interpret the process output files to yield a test status."""
         exit_code = outputs["exit_code"].read_text()
         exit_code = exit_code.strip()
-        if exit_code == "0":
+        if exit_code == self.pass_code:
             status = TestStatus.PASS
         else:
             status = TestStatus.FAIL
         return status
 
+    # TODO: make changes to this package or the nextflow workflow so that file mounting is handled cleaner
+    def short_string_path(self, path: Path, substring: str) -> str:
```

Comment:
Self isn't used, so can this be a staticmethod? Can we also make this a "hidden/helper" function with _ in the beginning?

Alternatively, we could put this in a utility module but I see why this is here.

Reply:
changed to a staticmethod starting with "_" in the latest commit.

I considered putting this in a utility module, but I figured that while it is only used by external tests it should stay here.